src/journal/journald-native.c

```diff
@@ -88,7 +89,7 @@ void server_process_native_message(
                 const char *label, size_t label_len) {
 
         struct iovec *iovec = NULL;
-        unsigned n = 0, j, tn = (unsigned) -1;
+        unsigned n = 0, j, tn = -1;
```

Comment:
hmm, i must say, it's kinda strange that gcc doesn't complain if you assign a negative constant to an unsigned variable literally. But even if it doesn't complain, it appears right to me to keep the cast in there, to clarify that this is an intended mismatch of type and value